.github/workflows/pr.yml

```diff
@@ -470,14 +494,28 @@ jobs:
 
       # For faster/better builds of sdists.
       - run: python -m pip install wheel
-        shell: bash
+        shell: bash -l {0}
+        if: matrix.test-suite == 'integration' || matrix.test-suite == 'notebook' || matrix.test-suite == 'conda'
+
+      - name: Install Python Libs (python)
+        shell: bash -l {0}
+        run: |
+          python -m pip --disable-pip-version-check install -t ./pythonFiles/lib/python --no-cache-dir --implementation py --no-deps --upgrade -r ./requirements.txt
         if: matrix.test-suite == 'integration' || matrix.test-suite == 'notebook'
 
+      - name: Install Python Libs (conda)
+        shell: bash -l {0}
+        run: |
+          python -m pip --disable-pip-version-check install -t ./pythonFiles/lib/python --no-cache-dir --implementation py --no-deps --upgrade python-jsonrpc-server==0.4.0
+          python -m pip --disable-pip-version-check install -t ./pythonFiles/lib/python --no-cache-dir --implementation py --no-deps --upgrade ujson
+        if: matrix.test-suite == 'integration' || matrix.test-suite == 'notebook' || matrix.test-suite == 'conda'
+
+      # This step is slow.
```

Comment:
If it's slow might be ways we can cache it, I think core extension might have been doing that with python libs.

Reply:
a comment from their repo, they may have removed this comment since.
We are caching as well (better).